messagecard.go

```diff
@@ -143,12 +143,10 @@ type MessageCard struct {
 // MessageCard. Validation is performed to reject invalid values with an error
 // message.
 func (mc *MessageCard) AddSection(section ...*MessageCardSection) error {
-
 	for _, s := range section {
-
 		// bail if a completely nil section provided
 		if s == nil {
-			return fmt.Errorf("AddSection: nil MessageCardSection received")
+			return fmt.Errorf("func AddSection: nil MessageCardSection received")
```

Comment:
@dasrick Do you feel that all function references in error messages like this one should have the `func` prefix? I hadn't really thought about it until now.